### ISSUE TITLE
.pyi files inhibit docstrings from .py files from showing when hovering

### ISSUE THREAD

**Issue by YodaEmbedding** [microsoft/pyright#2135]
**Describe the bug**
Docstrings from source (.py files) do not display when type stubs (.pyi files) exist.

**To Reproduce**
Install Python 3.9+.
```python
import os
os.mkdir()
```
Attempt to obtain docstring of `mkdir` by hovering.

**Expected behavior**
Docstrings should appear when hovering.

**VS Code extension or command-line**
Neovim 0.5 built-in LSP.

**Additional context**
As more projects adopt .pyi files (e.g. Python standard library) which do not contain/mirror docstrings from source .py files, viewing documentation information about the code from within the editor becomes more and more difficult.

**Comment by erictraut** [comment]
Pylance, which is a language server built on top of the functionality in pyright, offers this feature. It merges doc strings from source files and associates it with type information from type stubs. We don't have any plans to port this functionality to pyright, which is primarily focused on type checking functionality. So if you want this functionality, you'll need to switch to pylance, which works only with VS Code.
